CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -35,6 +35,7 @@ function Staff:Staff(...)
   self.parcelNr = 0
   self.leave_sounds = {}
   self.leave_priority = AnnouncementPriority.High
+  self.max_salary = 2000
```

Comment:
Question: Why is "wage" in `self.profile`, while max salary is in `self`?

ie is `self` the right spot?

Reply:
Shouldn't max salary depend on the job of the staff member?

Probably out of scope here but the config file also has same salary increase settings.

Reply:
btw I don't agree with the hard limit, it makes writing a level where eg doctors are paid huge sums of money impossible.

Not something that has to be done now though.

Reply:
max_salary doesn't belong here. It is not an instant variable of staff and we have many staff so it will be copied around a lot. config would be better.

Reply:
Will move to config